submissions/mayowaShoyinka/htmlForms/signup.html

```diff
@@ -7,52 +7,65 @@
     <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&family=Raleway:ital,wght@1,300&family=Roboto:ital,wght@1,300&display=swap" rel="stylesheet">
 </head>
 <body>
-	<!--------------- Header Starts Here--------------->
-    <div class="navigation-header">   
+    <!--------------- Header Starts Here--------------->
+    <div class="navigation-header">
         <img src="../html/img/logo-1.png" class="logo" alt="company-logo">
-        <div class="navigation-header-link">  
+        <div class="navigation-header-link">
             <a href="#">Product page</a>
             <a href="#">Contact Us</a>
             <a href="signup.html">Sign-up</a>
         </div>
     </div>
  <!----------------- Body Starts Here--------------->
 <section id="form-container"> 
-			<!--form-left -->
-			<div class="form-left">
-				<h1>Register your <br> Account</h1>
-				<p>Registering an account gives you access to <br> sweet promo and events available to only registered customers</p>
-				<h4>Already have an account? <a href="#">Sign in </a></h4>
-			</div>
-			<!--form-right -->
-			<div class="form-right">
-				<form action="">
-					<label for="firstname">First Name</label><br>
-					<input type="text" name="firstname"><br>
-					<label for="lastname">Last Name</label><br>
-					<input type="text" name="lastname"><br>
-					<label for="othername">Other Name</label><br>
-					<input type="text" name="othername"><br>
-					<label for="phone">Phone</label><br>
-					<input type="text" name="phone"><br>
-					<label for="email">Email Address</label><br>
-					<input type="email" name="email-address"><br>
-					<label for="password">Password</label><br>
-					<input type="password" name="password"><br>
-					<label for="lastname">Confirm Password</label><br>
-					<input type="password" name="Cpassword"><br>			
-					<input type="checkbox" class="checkbox">
-					<label for="t-c">I agree to the Terms and Conditions</label>
-					<br>
-					<button type="button">SIGN UP</button>
-				</form>	
-			</div>			
-		</div>
-	</div>	
+            <!--form-left -->
+            <div class="form-left">
+                <h1>Register your <br> Account</h1>
+                <p>Registering an account gives you access to <br> sweet promo and events available to only registered customers</p>
```

Comment:
I will suggest you resize the width of `form-left` div instead of adding a `br` tag into your h1 and p tag

Reply:
resize in what way, i thought this section has been viewed during html and css: and i was not told to adjust it. Thanks